### PR TITLE
Update default profiles configmap to carry k8s version info for rke profiles

### PR DESCRIPTION
**Pull request by prachidamle**
Carry information about multiple rke profiles according to k8s major versions supported.

https://github.com/rancher/cis-operator/issues/84

### CODE REVIEW

**Review by mrajashree — APPROVED** [review]

**Review by brendarearden — APPROVED** [review]
LGTM

File: packages/rancher-cis-benchmark/charts/templates/configmap.yaml [hunk 1]
```diff
@@ -5,7 +5,9 @@ metadata:
   namespace: {{ template "cis.namespace" . }}
 data:
   # Default ClusterScanProfiles per cluster provider type
-  rke: "rke-profile-permissive-1.6"
+  rke: |-
+    <1.16.0: rke-profile-permissive-1.5
```
Comment (dramich) [inline packages/rancher-cis-benchmark/charts/templates/configmap.yaml:9]: Is the space after the colon needed for something? If not we should drop it

Comment (prachidamle) [reply packages/rancher-cis-benchmark/charts/templates/configmap.yaml:9]: No, I added it for readability

Comment (dramich) [reply packages/rancher-cis-benchmark/charts/templates/configmap.yaml:9]: OK, that sounds fine then.

**Review by dramich — APPROVED** [review]

**Comment by prachidamle** [comment]
updated the image tag for cis-operator

**Review by dramich — APPROVED** [review]

**Pull request merged**